govcd/edgegateway.go

```diff
@@ -43,14 +43,14 @@ func NewEdgeGateway(cli *Client) *EdgeGateway {
 // Struct which covers NAT rule fields
 type NatRule struct {
 	natType      string
```

Comment:
You capitalized all fields of this struct, but left this one poor guy lower case :)

Reply:
This is done intentionally. This field is set by specific functions.

Reply:
Which specific functions? Why does it have to be "different"?

Reply:
AddSNat and AddDNAT - you have pass to different type of rule to API

Reply:
```

Reply:
Made public, cause you still can access AddNATRuleAsync as it's public, so field still have to be setted.

Reply:
Thanks!